Podfile.lock

```diff
@@ -256,12 +265,13 @@ SPEC CHECKSUMS:
   SVProgressHUD: 1428aafac632c1f86f62aa4243ec12008d7a51d6
   UIDeviceIdentifier: a959a6d4f51036b4180dd31fb26483a820f1cc46
   WordPress-Aztec-iOS: 448c4ba1c7d748e9ed881f328963ccdc8977e0d8
-  WordPressShared: b1bed3aa4db4438fd9fe78ccd0be58c8fee19c17
+  WordPressKit: 0e6bd2cd98a70f828fed793b9edea8a999ff7ff8
+  WordPressShared: 13a743d923796382419e03521e47491e9322087e
   WordPressUI: 17ea58d8b73d9f6683fe1fa71b31a7734d215e39
   WPMediaPicker: ceb613e43eae03268e73835d48d67f92f595aca6
   wpxmlrpc: bfc572f62ce7ee897f6f38b098d2ba08732ecef4
   ZendeskSDK: 2cda4db2ba6b10ba89aeb8dddaa94e97c85946a0
 
-PODFILE CHECKSUM: 6f9992e4e9c975b84ebd7c7ed9deba3c19d2a625
+PODFILE CHECKSUM: 2395b82e93e6acd063655260daafff664f577084
 
-COCOAPODS: 1.5.2
```

Comment:
Why are we going back on the version of Cocoapods?

Reply:
Because i had no idea 1.5.2 was out! updating, thanks for the heads up!!